src/category_theory/subobject/well_powered.lean

```diff
@@ -57,10 +57,27 @@ theorem well_powered_of_essentially_small_mono_over
   well_powered C :=
 { subobject_small := λ X, (essentially_small_mono_over_iff_small_subobject X).mp (h X), }
 
+section
 variables [well_powered C]
 
 instance essentially_small_mono_over (X : C) :
   essentially_small.{v} (mono_over X) :=
 (essentially_small_mono_over_iff_small_subobject X).mpr (well_powered.subobject_small X)
 
+end
+
+section equivalence
+variables {D : Type u₂} [category.{v} D]
+
+private theorem well_powered_congr_aux (e : C ≌ D) (h : well_powered C) : well_powered D :=
```

Comment:
Perhaps this should be a public lemma, with `h` as a typeclass argument? That way we can say, eg,
```
instance : well_powered my_cat := well_powered_congr_aux equiv_with_other_cat
```